spec/std/http/cookie_spec.cr

```diff
@@ -194,36 +194,80 @@ module HTTP
 
       it "parse max-age as seconds from current time" do
```

Comment:
I guess this description should follow-up with your changes.

Reply:
Good catch. I've updated it.

Reply:
Actually I thought about the `from current time` part ;)

Reply:
Oops, yeah you're right :joy: